src/set_theory/game/nim.lean

```diff
@@ -0,0 +1,257 @@
+/-
+Copyright (c) 2020 Fox Thomson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Fox Thomson
+-/
+import set_theory.game.impartial
+import set_theory.ordinal
+import data.set
+import logic.basic
+
+universes u v
+
+/-!
+# Nim and the Sprague-Grundy theorem
+
+This file contains the defition for nim for any ordinal `O`. In the game of `nim O₁` both players
```

Comment:
```suggestion
This file contains the definition for nim for any ordinal `O`. In the game of `nim O₁` both players
```

Reply:
There are quite a few spelling issues in the doc-strings. Perhaps you can skim through and catch them all, then resolve this conversation?